### PR TITLE
Update sponsors

### PR DESCRIPTION
**Pull request by dnahol**
Add sponsor logos to Supporters page https://github.com/girldevelopit/gdi-website/issues/527
This is my first contribution here so let me know if I missed anything!

screen shot 2017-01-31 at 6 04 27 pm

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -10,6 +10,9 @@ Vagrant.configure(2) do |config|
   # For a complete reference, please see the online documentation at
   # https://docs.vagrantup.com.
 
+  # ssh agent forwarding
+  config.ssh.forward_agent = true
```
Comment (kstack7) [inline Vagrantfile:14]: Can you explain the reason for this change?

**Comment by kstack7** [comment]
Thanks so much @dnahol :) Just added one comment regarding an addition just so we have a constant flow of communication when adding/removing code here :) Let's others pick up and learn through contributing to the site...

**Review by kstack7 — COMMENTED** [review]
Hey there @dnahol -- can you check out my comment in your code? Curious what your thoughts were behind adding the forwarding.

File: Vagrantfile (hunk 1 above)
Comment (dnahol) [reply Vagrantfile:14]: Hi Kristen. Sorry, I just saw this now. So the reason for this change was that in order to use the Vagrant environment I kept getting error messages saying my keys were not verified and I couldn't push (or even fetch). So I got some outside advice to do ssh agent forwarding to fix this issue. This did fix the issue and I was able to push the commit.

The error:
Dalias-MacBook-Pro:gdi-website dalia$ git fetch upstream
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Pull request merged**